base/ltoutenc.dtx

```diff
@@ -3601,7 +3603,7 @@
 \xdef\@fontenc@load@list{\@fontenc@load@list
   \@elt{\csname opt@fontenc.sty\endcsname}}
 %    \end{macrocode}
-%    
+%
```

Comment:
thanks for including this -- we should purge all trailing whitespace! :)

Reply:
Hm. I don't think that it was me, so either git or windows should get the praise.